panc/src/test/pan/Functionality/choice/choice20.pan

```diff
@@ -0,0 +1,9 @@
+#
+# @expect=org.quattor.pan.exceptions.ValidationException
```

Comment:
is there a way to distinguish the exception between this failure and the on ein choice19?